### PR TITLE
add changelog

### PR DESCRIPTION
**Pull request by dineshyv**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+- Persistence for registered objects with distribution
+- Ability to persist memory banks created for FAISS
+- PostgreSQL KVStore implementation
+- Environment variable placeholder support in run.yaml files
+- Provider deprecation support
+- Comprehensive Zero-to-Hero notebooks and quickstart guides
+- Colab notebook integration for Llama Stack lesson
+- Remote::vllm provider with vision model support
+- Support for quantized models in Ollama
+- Vision models support for Together, Fireworks, Meta-Reference, and Ollama
```
Comment (ashwinb) [inline CHANGELOG.md:16]: can you combine 14 and 16?

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+- Bedrock distribution with safety shields support
+
+### Changed
+- Split safety into distinct providers (llama-guard, prompt-guard, code-scanner)
+- Improved distribution registry with SQLite default storage
+- Enhanced test infrastructure with composable fixtures
```
Comment (ashwinb) [inline CHANGELOG.md:28]: is this really what happened in 0.0.52 -> 0.0.53?

File: CHANGELOG.md (hunk 1 above)
Comment (dineshyv) [reply CHANGELOG.md]: done.

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+- Comprehensive Zero-to-Hero notebooks and quickstart guides
+- Colab notebook integration for Llama Stack lesson
+- Remote::vllm provider with vision model support
+- Support for quantized models in Ollama
+- Vision models support for Together, Fireworks, Meta-Reference, and Ollama
+- Bedrock distribution with safety shields support
+
+### Changed
+- Split safety into distinct providers (llama-guard, prompt-guard, code-scanner)
+- Improved distribution registry with SQLite default storage
```
Comment (ashwinb) [inline CHANGELOG.md:27]: nuke this or combine with persistence for registered objects bullet above

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+- Changed provider naming convention (`impls` → `inline`, `adapters` → `remote`)
+- Updated API signatures for dataset and eval task registration
+- Restructured folder organization for providers
+- Enhanced Docker build configuration
+- Added version prefixing for REST API routes
+
+### Removed
+- `llama stack configure` command
+
+### Fixed
```
Comment (ashwinb) [inline CHANGELOG.md:41]: "Bug Fixes" maybe?

File: CHANGELOG.md (hunk 3 above)
Comment (dineshyv) [reply CHANGELOG.md:27]: done.

File: CHANGELOG.md (hunk 2 above)
Comment (dineshyv) [reply CHANGELOG.md:28]: i dont think so. removing this.

File: CHANGELOG.md (hunk 4 above)
Comment (ashwinb) [reply CHANGELOG.md:41]: also some of the line items here are rather puny and should be nuked. I think the Changelog should be brief and have important things only.

File: CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+- Agent system instructions persistence after first turn
+- PostgreSQL config validation
+- vLLM adapter chat completion signature
+- Routing table lookup key for memory banks
+- Together inference validator
+- Exception handling for SSE connection closure
+
+### Development
+- Added new test migration system
```
Comment (ashwinb) [inline CHANGELOG.md:50]: what does this mean?

Comment (dineshyv) [reply CHANGELOG.md:48]: removed the development and fixes section since its not really useful.

File: CHANGELOG.md (hunk 4 above)
Comment (dineshyv) [reply CHANGELOG.md:39]: agree. removed the development section and

**Review by ashwinb — APPROVED** [review]

**Pull request merged**